release-notes/unreleased/IGAPP-983-wrong-text-alignment.yml

```diff
@@ -0,0 +1,5 @@
+issue_key: IGAPP-983
+show_in_stores: false
+platforms:
+  - web
```

Comment:
Created https://issues.tuerantuer.org/browse/IGAPP-1079 since I don't know how to solve this (apart from content rendered as html which I fixed in this PR). If you have any ideas how to solve it for native it would be appreciated.